ci/run-benchmarks.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPT_DIR="$(dirname "$0")"
+
+source "$SCRIPT_DIR/common-env"
+
+# NOTE(thomastaylor312): We have to specify the image ID, so this is the command you can use to the
+# find the latest images and select the one compatible with the k8s version
+# oci ce node-pool-options get --compartment-id "$COMPARTMENT_ID" --node-pool-option-id all
+export OCI_MANAGED_NODE_IMAGE_ID="ocid1.image.oc1.iad.aaaaaaaarkexs7ijdqffxvy6dyzepizvuzm25zijwp2nkv3nifbcvywzufsq"
```

Comment:
Should we set this in the pipeline / as a GitHub secret so that we can update the managed image without updating the script? I'm not 100% sure so let me know if I'm thinking about it wrong

Reply:
This isn't a bad idea. Can default to this and allow it to be overriden